README.md

```diff
@@ -70,4 +70,35 @@ By definition, members work on their respective squad, although they are free to
 
 ## 4. New hires checklist
 
-// Work in progress ⚠️
+Prior to starting, make sure you have a Babylon GitHub account and that you have access to the following repositories:
+
+- [babylon-ios](https://github.com/Babylonpartners/babylon-ios)
+- [ios-charts](https://github.com/Babylonpartners/ios-charts)
+- [ios-private-podspecs](https://github.com/Babylonpartners/ios-private-podspecs)
+- [ios-build-distribution](https://github.com/Babylonpartners/ios-build-distribution)
+- [ios-fastlane-match](https://github.com/Babylonpartners/ios-fastlane-match)
+
+Here's how to get the iOS project up and running.
+
+- [ ] Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios
```

Comment:
I'd remove the checkboxes as I don't think they serve any purpose for a public page. A list would be enough?